### ISSUE TITLE
Crash with this sentence

### ISSUE THREAD

**Issue by larseggert** [languagetool-org/languagetool#6262]
```echo "It is much faster than previous digital signature schemes, so ESIGN can be implemented on smart cards without special co-processors." | languagetool -```

causes this crash (with LanguageTool version 5.6 (2022-01-03 18:14:04 +0000, e1a2756):

```No language specified, using English (no spell checking active, specify a language variant like 'en-GB' if available)
Working on STDIN...
Exception in thread "main" java.lang.RuntimeException: java.util.concurrent.ExecutionException: java.lang.RuntimeException: java.lang.RuntimeException: Could not check sentence (language: English): <sentcontent>It is much faster than previous digital signature schemes, so ESIGN can be implemented on smart cards without special co-processors.
</sentcontent>
	at org.languagetool.MultiThreadedJLanguageTool.performCheck(MultiThreadedJLanguageTool.java:217)
	at org.languagetool.JLanguageTool.checkInternal(JLanguageTool.java:1002)
	at org.languagetool.JLanguageTool.check(JLanguageTool.java:929)
	at org.languagetool.JLanguageTool.check(JLanguageTool.java:914)
	at org.languagetool.commandline.CommandLineTools.checkText(CommandLineTools.java:108)
	at org.languagetool.commandline.CommandLineTools.checkText(CommandLineTools.java:84)
	at org.languagetool.commandline.Main.runOnFile(Main.java:192)
	at org.languagetool.commandline.Main.main(Main.java:464)
Caused by: java.util.concurrent.ExecutionException: java.lang.RuntimeException: java.lang.RuntimeException: Could not check sentence (language: English): <sentcontent>It is much faster than previous digital signature schemes, so ESIGN can be implemented on smart cards without special co-processors.
</sentcontent>
	at java.base/java.util.concurrent.ForkJoinTask.get(ForkJoinTask.java:1006)
	at org.languagetool.MultiThreadedJLanguageTool.performCheck(MultiThreadedJLanguageTool.java:214)
	... 7 more
Caused by: java.lang.RuntimeException: java.lang.RuntimeException: Could not check sentence (language: English): <sentcontent>It is much faster than previous digital signature schemes, so ESIGN can be implemented on smart cards without special co-processors.
</sentcontent>
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at java.base/java.util.concurrent.ForkJoinTask.getThrowableException(ForkJoinTask.java:600)
	... 9 more
Caused by: java.lang.RuntimeException: Could not check sentence (language: English): <sentcontent>It is much faster than previous digital signature schemes, so ESIGN can be implemented on smart cards without special co-processors.
</sentcontent>
	at org.languagetool.JLanguageTool$TextCheckCallable.getOtherRuleMatches(JLanguageTool.java:2026)
	at org.languagetool.JLanguageTool$TextCheckCallable.call(JLanguageTool.java:1901)
	at org.languagetool.MultiThreadedJLanguageTool.lambda$performCheck$1(MultiThreadedJLanguageTool.java:200)
	at java.base/java.util.concurrent.ForkJoinTask$AdaptedCallable.exec(ForkJoinTask.java:1448)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)
Caused by: java.lang.NullPointerException
	at org.languagetool.rules.en.CompoundRule.isMisspelled(CompoundRule.java:162)
	at org.languagetool.rules.AbstractCompoundRule.isCorrectSpell(AbstractCompoundRule.java:255)
	at org.languagetool.rules.AbstractCompoundRule.filterReplacements(AbstractCompoundRule.java:176)
	at org.languagetool.rules.AbstractCompoundRule.match(AbstractCompoundRule.java:150)
	at org.languagetool.JLanguageTool.checkAnalyzedSentence(JLanguageTool.java:1409)
	at org.languagetool.JLanguageTool.access$1500(JLanguageTool.java:77)
	at org.languagetool.JLanguageTool$TextCheckCallable.getOtherRuleMatches(JLanguageTool.java:1986)
	... 8 more
```

**Comment by danielnaber** [comment]
Thanks for the report. This only seems to happen with automatic language detection. Using `--language en-US` makes it work.

**Comment by jaumeortola** [comment]
This bug was fixed some days ago (https://github.com/languagetool-org/languagetool/commit/ef3e158f0791d73c5a8a6474fcf425cc3cf9a3a2). The fix is in 5.7-SNAPSHOT.
